testscript/exe.go

```diff
@@ -211,3 +211,5 @@ func (nopTestDeps) WriteHeapProfile(io.Writer) error {
 	// Not needed for Go 1.10.
 	return nil
 }
+
+func (nopTestDeps) SetPanicOnExit0(bool) {}
```

Comment:
Worth a comment on why this is here, especially given we don't have a 1.16 build to break?

Reply:
ack, will do